src/lib.rs

```diff
@@ -66,15 +69,31 @@ use {
   prometheus_metrics::serve_prometheus,
 };
 
+#[derive(Parser, Debug)]
+struct CmdArgs {
+  #[arg(long, default_value_t = false)]
+  /// if you start multiple lemmy server instances set this to true on all but one of them
+  disable_scheduled_tasks: bool,
+  /// set to false to disable the http server
+  #[arg(long, default_value_t = true, action=ArgAction::Set)]
+  http_server: bool,
+  /// set to false to disable the outgoing federation in this process
+  #[arg(long, default_value_t = true, action=ArgAction::Set)]
+  send_activities: bool,
+  /// the index of this outgoing federation process (1-based). only useful if you want to split federation work into multiple servers
```

Comment:
What do you mean by 1-based?

Reply:
It means the first worker should have this number set to `1` not `0`. I've updated the doc comment to be clearer.